spec/lib/sem/cli/teams_spec.rb

```diff
@@ -455,9 +459,10 @@
 
       context "org names are not matching" do
         it "raises an exception" do
-          expect { sem_run("teams:shared-configs:remove rt/developers org/tokens") }.to raise_exception(
-            Sem::Errors::OrgNamesNotMatching
-          )
+          stdout, stderr = sem_run("teams:shared-configs:remove rt/developers org/tokens")
+
+          expect(stderr.strip).to eq(Sem::Views::Teams.org_names_not_matching)
+          expect(stdout.strip).to eq("")
```

Comment:
Let's check the exit status as well. I think there are some examples of checking it in the CLI.